implementations/contracts/ERC725/ERC725X.sol

```diff
@@ -16,17 +13,16 @@ import "@openzeppelin/contracts/access/Ownable.sol";
  *
  *  @author Fabian Vogelsteller <fabian@lukso.network>
  */
-contract ERC725X is Ownable, ERC725XCore {
+contract ERC725X is ERC725XCore {
     /**
      * @notice Sets the owner of the contract
      * @param _newOwner the owner of the contract.
      */
     constructor(address _newOwner) {
         // This is necessary to prevent a contract that implements both ERC725X and ERC725Y to call both constructors
         if (_newOwner != owner()) {
-            transferOwnership(_newOwner);
+            Ownable.initOwner(_newOwner);
```

Comment:
We need to change the name of the contract to avoid contract name conflicts.
Please change the contract and here to:
`OwnableUnset`